.travis.yml

```diff
@@ -1,14 +1,95 @@
 sudo: required
-services:
-  - docker
+dist: trusty
 language: node_js
 node_js:
-  - "6.1"
+    - "6"
+## home folder is /home/travis/radanalyticsio/oshinko-webui
+services:
+- docker
+matrix:
+  include:
+    - env: TO_TEST=e2e
+  fast_finish: true
+
 before_install:
-  - docker build -t radanalyticsio/oshinko-webui .
+## add insecure-registry and restart docker
+- |
+  if [ "$TO_TEST" = "e2e" ]; then
```

Comment:
i think we can remove this if condition.
i have a feeling that this causing the "view config" in travis-ci to fail somehow.